### PR TITLE
css-brochure-form | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
image

I'm having trouble moving the last few things around.
This is what I have so far.

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude!! I know you know there's still things to fix, I'm just putting in a review so you know it still needs to be addressed.

**Comment by rocky-truong** [comment]
image


I was able to move the things to the left, but I think there was an easier way.

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#fullname {
+  margin-right: 400px;
+}
+
+#wherefor {
+  margin-right: 280px;
+}
+
+#hiring {
+  margin-right: 25px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:56]: That's a lot of margin!

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#email-address {
+  margin-right: 330px;
+}
+
+#fullname {
+  margin-right: 400px;
+}
+
+#wherefor {
+  margin-right: 280px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:52]: That's a lot of margin!

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  margin-bottom: 10px;
+  margin-top: 5px;
+}
+
+#email-address {
+  margin-right: 330px;
+}
+
+#fullname {
+  margin-right: 400px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:48]: That's a lot of margin!

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  width: 450px;
+  border-radius: 4px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+  margin-top: 5px;
+}
+
+#email-address {
+  margin-right: 330px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:44]: That's a lot of margin!

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+button {
+  text-transform: uppercase;
+  color: white;
+  background-color: orangered;
+  font-family: 'Brandon Grotesque', sans-serif;
+  border-radius: 12px;
+  padding: 5px 20px;
+  margin-left: 350px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:30]: That's a lot of margin!

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great job so far, man! You have a lot of margin going on here, though. One way you can get around this is to set a width on the `form` element, and center the content either using `margin: auto` or flexbox. Once you confine the `form` content to a certain area, the movements you will need to make will be much smaller.

**Comment by rocky-truong** [comment]
image

Adding the flexbox to my form container helped centering the whole thing. So whether I made the window bigger or smaller on my browser, the form remained consistent in the middle.

I couldn't figure out how to force the labels to the left side. I've tried float: left;, align-text: left.. Neither of these worked.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude!! I know you still have this in progress, just wanted to give you a review so it's clear there are still changes needed. If you have any questions on how to tackle this, make sure you reach out in Slack!

**Comment by rocky-truong** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice job, dude!!

**Pull request merged**